### PR TITLE
Added methods section to document

### PR DESCRIPTION
**Pull request by Pankai222**

### CODE REVIEW

File: report/report.tex [hunk 1]
```diff
@@ -97,29 +97,24 @@
...
-\lettrine[nindent=0em,lines=3]{L} orem ipsum dolor sit amet, consectetur adipiscing elit.
-\blindtext % Dummy text
-
-\blindtext % Dummy text
+Voting systems set in place to identify helpful reviews help consumers circumvent this issue, yet these reviews still get lost in the sheer number of unlabeled reviews. Our machine learning model aims to further filter reviews by analyzing verbosity, sentiment and other aspects, to discover what makes a review helpful.
 
 %------------------------------------------------
 
 \section{Methods}
+We have made use of the agile and iterative research approach proposed in Nunamaker et al., to continuously gain feedback on the activities involved in our process.
```
Comment (nullxDEADBEEF) [inline report/report.tex:107]: Remember to reference the Nunamaker paper :DD

File: report/report.tex [hunk 2]
```diff
@@ -97,29 +97,24 @@
...
-\item Donec dolor arcu, rutrum id molestie in, viverra sed diam
-\item Curabitur feugiat
-\item turpis sed auctor facilisis
-\item arcu eros accumsan lorem, at posuere mi diam sit amet tortor
-\item Fusce fermentum, mi sit amet euismod rutrum
-\item sem lorem molestie diam, iaculis aliquet sapien tortor non nisi
-\item Pellentesque bibendum pretium aliquet
-\end{itemize}
-\blindtext % Dummy text
+Our research aims at creating a predictive model for identifying helpful Amazon reviews. Therefore, the data used in this study is a subset of Amazon product reviews divided into categories, with each product and reviewer having at least 5 reviews.
```
Comment (nullxDEADBEEF) [inline report/report.tex:109]: Is the 5-core used anymore?

File: report/report.tex [hunk 3]
```diff
@@ -97,29 +97,24 @@
...
+
+As our model is primarily trained on text data, common Natural Language Processing techniques have been used in the preprocessing of data and the subsequent structuring of this data into input data.
+
+For converting our text data into parametric data for use in building our model, we have chosen to use document-term matrices through Count Vectorization and TF-IDF Vectorization. By employing two methods of text vectorization, we can more easily determine which term weighing factor improves the most upon the performance metrics of our model.
+
+Since we are dealing with a binary classification problem due to separation of reviews into categories of helpful and unhelpful, we make use of several different classifiers in training our model. We are labelling each review on its helpfulness to make supervised learning through each of these classifiers possible.
 
-Text requiring further explanation\footnote{Example footnote}.
+For our model evaluation we make use of primarily two statistics. A confusion matrix is used in summarizing the performance of the classifier algorithms used in training our model. Youden's Index is then applied to make up for an unbalanced dataset.
```
Comment (nullxDEADBEEF) [inline report/report.tex:117]: Skal man skrive noget om Youden's Index fortaeller noget om modellens predictive power?

File: report/report.tex (hunk 1 above)
Comment (Pankai222) [reply report/report.tex:107]: To be done later 8)

File: report/report.tex (hunk 3 above)
Comment (Pankai222) [reply report/report.tex]: Done 🗡️

File: report/report.tex (hunk 2 above)
Comment (nullxDEADBEEF) [reply report/report.tex:114]: http://deepyeti.ucsd.edu/jianmo/amazon/index.html
Mener at vi bare har downloadet reviews filerne ikke 5-core, ihvertfald til at køre modellerne

Comment (Pankai222) [reply report/report.tex:114]: Nå, okay. Men det finder vi ud af i en anden section :D

Comment (nullxDEADBEEF) [reply report/report.tex:114]: Alrighty :D

File: report/report.tex (hunk 1 above)
Comment (nullxDEADBEEF) [reply report/report.tex:112]: Bare at vi husker det :DDD

**Comment by nullxDEADBEEF** [comment]
In another PR we need to check up on:
- the 5-core reviews
- referencing the Nunamaker paper

**Pull request merged**